SplunkAppForWazuh/appserver/static/js/config/routes/overview-states.js

```diff
@@ -61,11 +61,16 @@ define(['../module'], function(module) {
                   const currentExtensions = $currentDataService.getExtensions(
                     id
                   )
-                  const result = currentExtensions
-                    ? currentExtensions
-                    : $requestService.httpReq(`GET`, `/manager/extensions`)
-                  return await result
+                  const result = {};
```

Comment:
Modularize this logic into a function

Reply:
Done [here](https://github.com/wazuh/wazuh-splunk/pull/367/commits/f13b3412c291a73876ec08cd94eb813d4283bdf7).